<launch>
	<arg name="port" default="62345"/>
	<arg name="host" default="192.168.1.10" />
        <arg name="use_localdatacenter" default="false" />
	<arg name="map_name"/>
	<arg name="db_name" default="somadata" />
        <arg name="object_collection_name" default="object"/>
        <arg name="roi_collection_name" default="roi"/>

        <param name="mongodb_port" value="$(arg port)" />
	<param name="mongodb_host" value="$(arg host)" />
 

<node name="message_store" pkg="mongodb_store" type="message_store_node.py">
	<param name="mongodb_use_localdatacenter" value="$(arg use_localdatacenter)" />
</node>

<node name="soma_map" pkg="soma_map_manager" type="soma_map_manager_node.py" args="--mapname $(arg map_name)" output="screen"/>
	
<node name="soma_data_manager_node" pkg="soma_manager" type="data_manager_node.py" args="--object_db_name $(arg db_name) --object_collection_name $(arg object_collection_name)" output="screen"/>

<node name="soma_query_manager_node" pkg="soma_manager" type="soma_query_manager_node" args="$(arg db_name) $(arg object_collection_name) $(arg db_name) $(arg roi_collection_name)" output="screen"/>
	
   
</launch>
